src/services/chapter.ts

```diff
@@ -177,7 +175,6 @@ class ChapterParser {
 
     public static parseComments(html: string): ChapterComment[] {
         const $ = cheerio.load(html);
-
```

Comment:
as general advice: i would avoid these more or less random/opinionated changes like removing an empty line or something unless your mr is specifically about improving style

i don't care about it in this case

Reply:
was an accident